src/example-relay/src/SSRQueryRenderer.js

```diff
@@ -7,16 +7,14 @@ import { getDataFromRequest } from '@adeira/relay-utils';
 
 import createRelayEnvironment from './createRelayEnvironment';
 
-type RendererProps = {| +[key: string]: any |};
-
-type Props = {|
+type Props<T> = {|
   +query: GraphQLTaggedNode,
   +variables: { [name: string]: mixed, ... },
```

Comment:
Should we annotate the rest with this type as well? For example, we have something like this:

```ts
readonly render: (relayProps: TOperation['response']) => React.ReactNode;
readonly variables?: TOperation['variables'];
```

So for example, that correct variables are being taken into account.

Reply:
Yeah, I can have a look at it and see 👍

Reply:
I just imported type `Variables` type fro `@adeira/relay-runtime` here.

Reply:
Mm, that's not what I meant because it won't actually check the variables. You should take the actual `<T>` into account in my opinion. But OK - can be improved later if someone is interested in better types.